.github/workflows/translation_check.py

```diff
@@ -148,8 +150,11 @@ def count_translations(dir_with_translations, print_info, reference_file):
             print(f'\t{', '.join(messages["missing"])}')
             print(f'Unnecessary (not in en-GB) ({not_needed[lang]}):')
             print(f'\t{', '.join(messages["unnecessary"])}')
-            print(f'Unexpected (should not be translated) ({len(messages["unexpected"])}):')
+            print(f'Technical strings modified ({len(messages["unexpected"])}):')
             print(f'\t{', '.join(messages["unexpected"])}')
+            if len(messages["unexpected"]):
+                print(f'\tIt\'s not expected for these to be translated, but if it makes sense for your language,')
+                print(f'\tadd a waiver for them on suppress_warning.py and open a PR explaining why.')
```

Comment:
I wonder whether it will be confusing if "these" is referring to the strings above or below
```suggestion
                print(f'\tIt\'s not expected for the above to be translated, but if it makes sense for your language,')
                print(f'\tadd a waiver for them on suppress_warning.py and open a PR explaining why.')
```